README.md

```diff
@@ -50,6 +50,29 @@ relied upon.
 
 When using an emulator, you can press `cmd+m` to simulate a menu button press.
 
+### Building with API keys
+Certain services require an API key, so you'll need to build with the key to use them in the apk.
+
+1. To build with the API key (for services such as Sentry), add a `<project-dir>/.<service>_release`
+file with your key, for example, `<project-dir>/.sentry_dsn_release`
+
+* To enable Sentry on Debug builds, additionally replace the `DataUploadPreference.isEnabled`
```

Comment:
nit: these bullets look funky when rendered:

![screenshot_2018-05-15 liuche firefox-tv](https://user-images.githubusercontent.com/759372/40075267-c8702f3e-5830-11e8-841c-60dbb91b5668.png)

Did you mean:
```
1. Bullet
    1. Sub-bullet?
```